spec-insert/spec/_fixtures/expected_output/body_params_tables.md

```diff
@@ -0,0 +1,193 @@
+<!-- spec_insert_start
+api: indices.put_settings
+component: request_body_parameters
+-->
+## Request body
+
+The index settings to be updated.
+
+The request body is __required__. It is a JSON object with the following fields.
```

Comment:
Are request bodies always JSON objects? If not, do we need to call this out or can we switch it to what's in the template:

"The following table lists the available request body fields. The request body is _required_".

Reply:
It's a JSON object 90% of the time but an NDJSON (array of Jsons) the other time.

Reply:
Maybe we can change the description to this then:

"The request body is a JSON object and is required. The following table lists the available request body fields."

Thoughts?